docs/Installing.asciidoc

```diff
@@ -71,12 +58,17 @@ zypper ar -f obs://devel:openQA:Leap:<version>/openSUSE_Leap_$LEAP_VERSION devel
 
 As required change +LEAP_VERSION+ to the version of openSUSE Leap you have installed.
 
-Then you can install them using this command.
+=== Installation
+You can install the packages using these commands.
 
 [source,sh]
 -------------------------------------------------------------------------------
-# all openSUSE
-zypper install devel-openQA:openQA
+# openSUSE Leap 42.3+
+zypper in openQA
```

Comment:
I find it harder to read this way. Now every reader has to read three paragraphs, skipping over the repository section that does not apply for the personal choice. I recommend to revert this PR. The single line improvements, e.g. the s/install them/install the packages/ makes sense though

Reply:
your motivation sounds strange to me. Do we have a goal to provide interesting book where we care that reader will not sleep till the end? 
It is installation guide which contains some steps which not used by everybody but this is why you have paragraph names to decide which one to read

Reply:
"I know my Pappenheimer :)" --coolo, some time ago